### DISCUSSION TITLE
Help me to solve my issue with ffmpeg_kit_flutter

### DISCUSSION THREAD

**Discussion by Dev-Demons** [arthenica/ffmpeg-kit-test#41] [Q&A]
This is my issue
flutter: Loading ffmpeg-kit-flutter.
[ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: MissingPluginException(No implementation found for method getLogLevel on channel flutter.arthenica.com/ffmpeg_kit)

I added the following to pubspec.yaml

dependencies:
  flutter:
    sdk: flutter
  ffmpeg_kit_flutter: 5.1.0-LTS
  ffmpeg_kit_flutter_video: 5.1.0-LTS

flutter:
  plugin:
    platforms:
      android:
        package: com.arthenica.flutter.ffmpeg
        pluginClass: FlutterFFmpegPlugin
      ios:
        pluginClass: FlutterFFmpegPlugin

My test code is :

import 'package:ffmpeg_kit_flutter/ffmpeg_kit.dart';

    FFmpegKit.execute('-i file1.mp4 -c:v mpeg4 file2.mp4')
        .then((session) async {
      final returnCode = await session.getReturnCode();

      if (ReturnCode.isSuccess(returnCode)) {
        print("this is the success");
        // SUCCESS

      } else if (ReturnCode.isCancel(returnCode)) {
        // CANCEL
        print("this is the cancel");
      } else {
        // ERROR
        print("this is the fail");
      }
    });

what is the reason and what can I do to solve this issue?
Any suggestions?

**Comment by niccoreyes** [comment]
Added these permissions in AndroidMainfest.xml

<uses-permission android:name="android.permission.INTERNET" />
<uses-permission android:name="android.permission.READ_INTERNAL_STORAGE" />
<uses-permission android:name="android.permission.WRITE_INTERNAL_STORAGE" />
<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
<uses-permission android:name="android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>



And went for flutter clean (not sure if this helps)

This is adding permissions for your app to read and write to storage especially for android. I don't know about other platforms though if they need something like this
